scripts/travis/build.sh

```diff
@@ -34,6 +35,12 @@ SCRIPTPATH="$( cd "$(dirname "$0")" ; pwd -P )"
 OS=$("${SCRIPTPATH}/../ostype.sh")
 ARCH=$("${SCRIPTPATH}/../archtype.sh")
 
+if [[ "${ARCH}" != "arm" ]]; then
+    curl -sL -o ~/gimme https://raw.githubusercontent.com/travis-ci/gimme/master/gimme
+    chmod +x ~/gimme
+    eval $(~/gimme $("${SCRIPTPATH}/../get_golang_version.sh"))
+fi
+
```

Comment:
could you move this code into the `scripts/travis/configure_dev.sh` so that it would get attempted multiple times ?
( travis sometimes has some temporary issues, which were proved to get solved by trying few times )

Reply:
@tsachiherman the gimme script will not configure the environment correctly from a subshell.

Reply:
so.. use the `ln` trick that I used.. it will make it working correctly, subshell or not..

Reply:
Using symbolic links only works when go is already installed, and the location of the links vary based on the platform.  One advantage of using the environment context is that we can now test multiple versions of go in parallel.